05-scientific-notebooks.Rmd

```diff
@@ -181,9 +181,11 @@ For [more about using Jupyter notebooks see this](https://realpython.com/jupyter
 
 <details> <summary>**Set up an R notebook**</summary>
 
-1. Start a new notebook by going to `File` > `New Files` > `R Notebook`.
+1. Start a new notebook by going to `File` > `New File` > `R Notebook`.
 2. Then open up this chapter’s example code folder and open the `make_heatmap.R` file.
-<img src ="https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE/export/png?id=1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE&pageid=gfaa026a583_0_13" alt="In RStudio, you can create a new notebook by going to File > New Files > R Notebook. Then open up this chapter’s example code folder and open the make_heatmap.R file." style="display: block; margin: auto;" />
+
+<img src ="https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE/export/png?id=1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE&pageid=gfaa026a583_0_13" alt="In RStudio, you can create a new notebook by going to File > New File > R Notebook. Then open up this chapter’s example code folder and open the make_heatmap.R file." style="display: block; margin: auto;" />
+
```

Comment:
Note that in my browser, most of the instructions for setting up the R notebook (points 3 - 8) were hidden (under the image?) although I could see them in the source code. I am hoping they will appear when I add newlines around the <img> code block to match formatting of python notebook section which is rendering correctly. But, this needs to be verified.